liberapay/constants.py

```diff
@@ -109,16 +109,16 @@ def with_vat(self):
 
 EVENTS = [
     Event('income', 1, _("When I receive money")),
-    Event('low_balance', 2, _("When there isn't enough money in my wallet to cover my donations")),
-    Event('withdrawal_created', 4, _("When a transfer to my bank account is initiated")),
-    Event('withdrawal_failed', 8, _("When a transfer to my bank account fails")),
-    Event('pledgee_joined', 16, _("When someone I pledge to joins Liberapay")),
-    Event('team_invite', 32, _("When someone invites me to join a team")),
-    Event('payin_bankwire_failed', 64, _("When a bank wire transfer to my Liberapay wallet fails")),
-    Event('payin_bankwire_succeeded', 128, _("When a bank wire transfer to my Liberapay wallet succeeds")),
-    Event('payin_bankwire_expired', 256, _("When a bank wire transfer to my Liberapay wallet expires")),
-    Event('payin_directdebit_failed', 512, _("When a direct debit from my bank account fails")),
-    Event('payin_directdebit_succeeded', 1024, _("When a direct debit from my bank account succeeds")),
+    Event('low_balance', 2, _("Money in my wallet can't cover donations")),
```

Comment:
This is a whole block of text to be narrowed down. The idea is to treat them as events, rather than explaining what makes for each event each time.
If a detailed description is really needed, a mouseover is the way to do it.